### PR TITLE
Remove support for Android WebView

### PR DESCRIPTION
**Pull request by jimmyjames**
### Changes

- Removes the `WebAuthActivity` class. Authentication will always use an external browser application.
- Removes the related `useBrowser` and `useFullscreen` flags.

### References

https://auth0.com/blog/google-blocks-oauth-requests-from-embedded-browsers/

### Checklist

- [X] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)

- [X] I have read the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)

- [X] All existing and new tests complete without errors

### CODE REVIEW

File: auth0/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -25,7 +25,4 @@
 <resources>
     <string name="com_auth0_webauth_permission_missing_title">Missing permissions</string>
     <string name="com_auth0_webauth_permission_missing_description">Some permissions required by this provider were not granted. You can try to authenticate again or go to the application\'s permission screen in the phone settings and grant them. The missing permissions are:\n%s</string>
-
-    <string name="com_auth0_webauth_network_error">Network error</string>
-    <string name="com_auth0_webauth_webview_loading_retry">Retry</string>
```
Comment (jimmyjames) [inline auth0/src/main/res/values/strings.xml:30]: These values were only used by the Web view activity, inspection identified these as now unused.

File: V2_MIGRATION_GUIDE.md [hunk 2]
```diff
@@ -45,6 +49,7 @@ Additionally, any classes that implemented `ParameterizableRequest` or `AuthRequ
 - The `com.auth0.android.util.Base64` class has been removed. Use `android.util.Base64` instead.
 - The `com.auth0.android.request.ParameterizableRequest` interface has been removed. The ability to add request headers and parameters has been moved to the `com.auth0.android.request.Request` interface.
 - The `com.auth0.android.request.AuthRequest` interface has been removed. The `com.auth0.android.request.AuthenticationRequest` interface can be used instead.
+- The `com.auth0.android.provider.WebAuthProvider` class has been removed. External browser applications will always be used for authentication.
```
Comment (lbalmaceda) [inline V2_MIGRATION_GUIDE.md:52]:
```suggestion
- The `com.auth0.android.provider.WebAuthActivity` class has been removed. External browser applications will always be used for authentication.
```

File: auth0/src/main/java/com/auth0/android/provider/AuthenticationActivity.java [hunk 3]
```diff
@@ -12,8 +12,6 @@
 public class AuthenticationActivity extends Activity {
 
     static final String EXTRA_USE_BROWSER = "com.auth0.android.EXTRA_USE_BROWSER";
```
Comment (lbalmaceda) [inline auth0/src/main/java/com/auth0/android/provider/AuthenticationActivity.java:14]: Please remove this flag, as there's no longer a decision to be made around what to use

**Review by lbalmaceda — CHANGES_REQUESTED** [review]

File: auth0/src/main/java/com/auth0/android/provider/AuthenticationActivity.java (hunk 3 above)
Comment (jimmyjames) [reply auth0/src/main/java/com/auth0/android/provider/AuthenticationActivity.java]: 👍 removed in a new commit

File: V2_MIGRATION_GUIDE.md (hunk 2 above)
Comment (jimmyjames) [reply V2_MIGRATION_GUIDE.md]: Fixed in a new commit

**Review by lbalmaceda — APPROVED** [review]

**Pull request merged**